### PR TITLE
Add UniversalAssetID and tests

### LINKED ISSUE #1168: [Swift] Implement universal asset identifier
https://developer.trustwallet.com/add_new_asset/universal_asset_id

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1168, remove duplicate code in TrustSDK and iOS

### CODE REVIEW

File: swift/Sources/Types/UniversalAssetID.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Copyright Trust Wallet. All rights reserved.
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
```
Comment (catenocrypt) [inline swift/Sources/Types/UniversalAssetID.swift:3]: TrustSDK -> Trust Wallet

**Review by catenocrypt — APPROVED** [review]

**Comment by catenocrypt** [comment]
Should we have this down in the common C++ layer?

**Comment by hewigovens** [comment]
@catenocrypt android doesn't need this right now

File: swift/Sources/Types/UniversalAssetID.swift (hunk 1 above)
Comment (hewigovens) [reply swift/Sources/Types/UniversalAssetID.swift:3]: good catch

**Comment by hewigovens** [comment]
And this is a lightweight type struct (in Types subspec), doesn't rely on cpp code

**Review by vikmeup — APPROVED** [review]

**Pull request merged**